tests/lib/readers/csv.test.js

```diff
@@ -13,75 +13,72 @@ beforeEach(() => {
 
   counts = {
     create: 0,
-    update: 0,
     delete: 0,
-    error: 0
+    error: 0,
+    update: 0,
   };
 });
 
 test('emits the expected number of events', () => {
+  const expected = {
```

Comment:
Not for this PR. I think we should consider writing this data to a file in test_data in the generate script so we don't have to keep updating the tests themselves.